src/events/ready.ts

```diff
@@ -1,12 +1,13 @@
+import { envParseBoolean } from '#lib/env';
 import { Slotmachine } from '#lib/games/Slotmachine';
 import { WheelOfFortune } from '#lib/games/WheelOfFortune';
 import { Events, Schedules } from '#lib/types/Enums';
-import { DEV, ENABLE_AUDIO, ENABLE_INFLUX, VERSION } from '#root/config';
 import { ApplyOptions } from '@sapphire/decorators';
 import { Event, EventOptions, Store } from '@sapphire/framework';
 import { blue, gray, green, magenta, magentaBright, red, white, yellow } from 'colorette';
 
-const style = DEV ? yellow : blue;
+const dev = process.env.NODE_ENV === 'development';
```

Comment:
already have a prop for this...and now they don't even check the same thing

Reply:
What do you mean?

Reply:
`Client#dev` checks that the env isn't `production`. Consistency plz

Reply:
Because NODE_ENV has 3 possible values, `production`, `development`, and `test`.

Inside pieces, they're exclusively either `production` or `development`, since Jest doesn't load them.

Reply:
Either way, addressed that in 75a226a.